src/general/NewGameSettings.cs

```diff
@@ -0,0 +1,676 @@
+        settings.FreeGlucoseCloud = freeGlucoseCloudButton.Pressed;
+
+        settings.MapType = MapTypeIndexToValue(mapTypeButton.Selected);
+
+        settings.IncludeMulticellular = includeMulticellularButton.Pressed;
+        settings.EasterEggs = easterEggsButton.Pressed;
+
+        // Stop music for the video (stop is used instead of pause to stop the menu music playing a bit after the video
+        // before the stage music starts)
+        Jukebox.Instance.Stop(true);
+
+        var transitions = new List<ITransition>();
+
+        if (Settings.Instance.PlayMicrobeIntroVideo && LaunchOptions.VideosEnabled)
+        {
+            transitions.Add(TransitionManager.Instance.CreateScreenFade(ScreenFade.FadeType.FadeOut, 1.5f));
+            transitions.Add(TransitionManager.Instance.CreateCutscene(
+                "res://assets/videos/microbe_intro2.ogv", 0.65f));
+        }
+        else
+        {
+            // People who disable the cutscene are impatient anyway so use a reduced fade time
+            transitions.Add(TransitionManager.Instance.CreateScreenFade(ScreenFade.FadeType.FadeOut, 0.2f));
+        }
+
+        TransitionManager.Instance.AddSequence(transitions, () =>
+        {
+            MainMenu.OnEnteringGame();
+
+            // TODO: Add loading screen while changing between scenes
+            var microbeStage = (MicrobeStage)SceneManager.Instance.LoadScene(MainGameState.MicrobeStage).Instance();
+            microbeStage.WorldSettings = settings;
+            SceneManager.Instance.SwitchToScene(microbeStage);
+        });
+    }
+
+    private void OnEnteringGame()
+    {
+        CheatManager.OnCheatsDisabled();
+        SaveHelper.ClearLastSaveTime();
+    }
+
+    private void OnDifficultyPresetSelected(int index)
+    {
+        // Set both buttons here as we only received a signal from one of them
+        difficultyPresetButton.Selected = index;
+        difficultyPresetAdvancedButton.Selected = index;
+
+        DifficultyPreset preset = SimulationParameters.Instance.GetDifficultyPresetByIndex(index);
+        settings.Difficulty = preset;
+
+        // If custom was selected, open the advanced view to the difficulty tab
+        if (preset.InternalName == custom.InternalName)
+        {
+            ChangeSettingsTab("Difficulty");
+            ProcessAdvancedSelection();
+            return;
+        }
+
+        mpMultiplier.Value = preset.MPMultiplier;
+        aiMutationRate.Value = preset.AIMutationMultiplier;
+        compoundDensity.Value = preset.CompoundDensity;
+        playerDeathPopulationPenalty.Value = preset.PlayerDeathPopulationPenalty;
+        glucoseDecayRate.Value = preset.GlucoseDecay * 100;
+        osmoregulationMultiplier.Value = preset.OsmoregulationMultiplier;
+        freeGlucoseCloudButton.Pressed = preset.FreeGlucoseCloud;
+
+        UpdateDifficultyPreset();
+    }
+
+    private void UpdateDifficultyPreset()
```

Comment:
```suggestion
    private void UpdateSelectedDifficultyPresetControl()
```
I was a bit confused what this method did at once, so I suggest a bit more descriptive name.